go.mod

```diff
@@ -32,50 +32,50 @@ require (
 	github.com/miekg/dns v1.1.35
 	github.com/modern-go/reflect2 v1.0.2
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
-	github.com/openshift/api v0.0.0-20220427141643-6e7069f2fba2
+	github.com/onsi/gomega v1.18.1
+	github.com/openshift/api v0.0.0-20220531073726-6c4f186339a7
```

Comment:
Sorry I missed this. This should line up with our apis submodule vendoring https://issues.redhat.com/browse/HIVE-1941

Reply:
https://github.com/openshift/hive/pull/1810 will enforce :)

Reply:
@abutcher done